packages/frontend/@n8n/composables/package.json

```
@@ -1,6 +1,7 @@
 {
 	"name": "@n8n/composables",
 	"type": "module",
+  "version": "1.0.0",
```

is this meant to be used by any external package? if not, maybe we should do this instead
```suggestion
  "private": true,
```